fixed-hash/src/hash.rs

```diff
@@ -51,7 +51,7 @@ macro_rules! construct_fixed_hash {
 	( $(#[$attr:meta])* $visibility:vis struct $name:ident ( $n_bytes:expr ); ) => {
 		#[repr(C)]
 		$(#[$attr])*
-		$visibility struct $name ([u8; $n_bytes]);
+		$visibility struct $name (pub [u8; $n_bytes]);
```

Comment:
I think this line is already part of the other PR, isn't it?

Reply:
yes, it is.